tests/NHSD.BuyingCatalogue.Testing.Data/EntityBuilders/SolutionEpicEntityBuilder.cs

```diff
@@ -21,8 +21,9 @@ public SolutionEpicEntityBuilder()
 
         public enum SolutionEpicStatus
         {
+            Undefined = 0,
             Passed = 1,
-            NotEvidenced = 3
+            NotEvidenced = 3,
```

Comment:
Did we have a status of 2 at some point?

Reply:
I think we do but I can only assume it's not used in the tests. I think there are a few things like this in BAPI – one for some further tech debit investigation at some point I think.

Reply:
Actually, the correct values are:

```
Passed = 1,
NotEvidenced = 2,
Failed = 3,
```

so I'm not sure why the values are as they are. I think I might add another tech debt task to look into these enum definitions in the test projects.

Reply:
Added tech debt task, 11129.